test/minitest_ext.rb

```diff
@@ -39,5 +40,11 @@ def to_s # :nodoc:
         ]
       end.join("\n")
     end
+
+    private
+
+    def stub_monorail_log_invocation
+      ShopifyCli::Git.stubs(:sha).returns("bb6f42193239a248f054e5019e469bc75f3adf1b")
```

Comment:
Does anyone recall why this was added? (The tests seem to pass without it). Was it maybe so that we can identify requests accidentally sent to Monorail during tests runs?

Reply:
I added it because all of our tests were going through a `Monorail.Log` call that then called `Git.sha`. If everything passes without it, then 🔥. Tests may have been reformatted since April to be better 😄